libs/extensions-registry/interfaces/property-editor-ui-extension-element.interface.ts

```diff
@@ -1,6 +1,6 @@
-import { DataTypePropertyPresentationModel } from '@umbraco-cms/backoffice/backend-api';
+import { UmbDataTypePropertyCollection } from '@umbraco-cms/backoffice/data-type';
 
 export interface UmbPropertyEditorExtensionElement extends HTMLElement {
 	value: unknown;
-	config: DataTypePropertyPresentationModel[];
+	config: UmbDataTypePropertyCollection;
```

Comment:
I think we should make the config optional. So you don't need to spin up an empty UmbDataTypePropertyCollection. And additionally, you would know when you got the config (not existing vs. getting the object) :-)

Reply:
Makes sense to me - there are a few places (mainly in WIP property editors) where we have a default empty array. I'll leave those for cleaning up when the editors are built, but have changed config to optional, so the prop doesn't need to exist at all.